generators/server/templates/pom.xml.ejs

```diff
@@ -105,11 +105,15 @@
         https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/${spring-boot.version} -->
         <liquibase.version>3.6.3</liquibase.version>
         <liquibase-hibernate5.version>3.6</liquibase-hibernate5.version>
+    <%_ if (devDatabaseType === 'h2Disk') { _%>
+        <h2.version>1.4.199</h2.version>
+    <%_ } _%>
+        <spring.version>5.1.5.RELEASE</spring.version>
```

Comment:
Why is this necessary? Shouldn't the version matches the version of spring-boot?

Reply:
It should, but unfortunately its mandatory due to liquibase-maven-pg dependency reference